packages/interpretations/src/components/Interpretations.js

```diff
@@ -5,6 +5,7 @@ import { LoadingMask } from '@dhis2/d2-ui-core';
 import { getFavoriteWithInterpretations } from '../models/helpers';
 import DetailsCard from './details/DetailsCard';
 import InterpretationsCard from './interpretations/InterpretationsCard';
+import _ from 'lodash';
```

Comment:
This adds the entire lodash library to the bundle. Please explicitly only import what you are using, e.g.:

```
import isEqual from 'lodash/fp/isEqual'
import pick from 'lodash/fp/pick'
```

This lets us use tree-shaking and dead code elimination to reduce the bundle size.

Reply:
Good point. Refactor this file and other 3 other with usages of lodash.

Reply:
Great!